share/src/main/java/io/zensoft/share/service/VacancyRetriever.java

```diff
@@ -0,0 +1,13 @@
+package io.zensoft.share.service;
+
+import io.zensoft.share.model.Vacancy;
+import io.zensoft.share.model.VacancyResponse;
+
+public interface VacancyRetriever {
+    /**
+     * the method is responsible for retrieving publish result of specific service
+     * @param vacancy the object given by {@link PublisherManagerService}
+     * @return {@link io.zensoft.share.model.VacancyResponse} object retrieved from DB
+     */
+    VacancyResponse getInfo(Vacancy vacancy);
```

Comment:
`getInfo` method should receive one more argument `PublisherServiceType`
like `VacancyResponse getInfo(Vacancy vacancy, PublisherServiceType publisherServiceType)` to be able to retrieve response for definite service

Reply:
You are right. Thank you for your comment, I'll fix this mistake now